apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -34,7 +34,7 @@ var canUpload = window.File || window.FileReader || window.FileList || window.Bl
 const profile = {
   name: 'fileExplorers',
```

Comment:
this new API will be public, so we should take the new very carefully. cc @bunsenstraat